yt_dlp/extractor/gedidigital.py

```diff
@@ -1,198 +1,105 @@
-import re
-
 from .common import InfoExtractor
-from ..utils import (
-    base_url,
-    determine_ext,
-    int_or_none,
-    url_basename,
-    urljoin,
-)
+from ..utils import int_or_none, join_nonempty, js_to_json, mimetype2ext
```

Comment:
IMO keep this as an import group for diff/maintenance reasons
```suggestion
from ..utils import (
    int_or_none,
    join_nonempty,
    js_to_json,
    mimetype2ext,
)
```

Reply:
`hatch fmt` automatically reverts this